utils/embeds.py

```diff
@@ -0,0 +1,33 @@
+from typing import List, Optional
+import discord
+
+class EmbedGenerator():
+    def __init__(self, entries: List, title: Optional[str] = "", description: Optional[str] = "", step: Optional[int] = 10):
+       self.entries = entries
```

Comment:
Personally, I like to add a line like `self.N = len(entries)` in my `__init__`s both for saving compute and brevity in the subsequent methods that might repeatedly require this value.